Dockerfile

```diff
@@ -25,6 +25,11 @@ RUN apt-get update && apt-get install -q -y \
       wget \
     && rm -rf /var/lib/apt/lists/*
 
+# install and run opensplice so tests will pass
+ENV RMW_IMPLEMENTATION=rmw_opensplice_cpp
+RUN apt-get update && apt-get install -q -y libopensplice69 \
```

Comment:
Installing `libopensplice69` looks fine here, as that is one of the [few packages omitted from the nightly](https://github.com/osrf/docker_images/blob/9ee7ec18d7168ae9763c9e6f7b5cc48b5370ec87/ros2/nightly/nightly/Dockerfile#L83). You could open a ticket if you'd like have it installed instead. I wouldn't be against it; just thought omitting non-default vendors could help keep the size down.